tools/migration/main.go

```diff
@@ -78,32 +78,36 @@ func main() { // nolint: deadcode
 	switch command {
 	case "-h", "--help":
 		showUsageAndExit(0)
-	case "describe", "buildonly", "migrate", "install":
-		logFile, err := openLogFile()
+	case "describe":
+		logger, err := newLoggerWithVerbose(true)
```

Comment:
I made this call because I tried to run describe and didn't like the behavior. I couldn't imagine why you'd ever have a script run it and care about the log content.  Since the instantiation of the migration runner requires a logger, it's created but it's ignored if describe is called. We could always log it if we decide to log describe output later.

Reply:
I understand that and believe it was the right call.  I was trying to ask whether it makes sense to remove the logger altogether from this case in main.

Reply:
i.e. make a noop logger by wiring up the logger to /dev/null or something when running describe.

Reply:
ah, right. Yes I considered this too but chose expedience.